src/app/components/forms/post/post.ts

```diff
@@ -392,12 +392,15 @@ export default class FormPost extends BaseForm<FormPostModel>
 			this.isSavedDraftPost = true;
 		}
 
+		// If the post is not a draft, default to posting to their profile.
```

Comment:
Yes, but why. Should be explaining like "Don't overwrite setting on a draft post because the user has already made a choice for this setting." or something like that.

Reply:
I like your wording more - updated